plugins/opener/src/reveal_item_in_dir.rs

```diff
@@ -193,4 +325,25 @@ mod imp {
 
         Ok(())
     }
+
+    pub fn reveal_items_in_dir(paths: &[PathBuf]) -> crate::Result<()> {
```

Comment:
From the implementations, it seems to me that we could change the original function `reveal_item_in_dir` to `reveal_items_in_dir` directly so we don't need much duplicated code like we currently have, since these are all internal, it's fine to change

Reply:
Updated, now there's just one fn with this logic under the hood